azext_edge/edge/providers/orchestration/work.py

```diff
@@ -137,12 +137,15 @@ def _format_instance_desc(self) -> str:
         return display_desc[:-1] + ""
 
     def _build_display(self):
-        pre_check_cat_desc = "Pre-Flight"
-        self._display.add_category(WorkCategoryKey.PRE_FLIGHT, pre_check_cat_desc, skipped=not self._pre_flight)
-        self._display.add_step(WorkCategoryKey.PRE_FLIGHT, WorkStepKey.REG_RP, "Ensure registered resource providers")
-        self._display.add_step(
-            WorkCategoryKey.PRE_FLIGHT, WorkStepKey.ENUMERATE_PRE_FLIGHT, "Enumerate pre-flight checks"
-        )
+        if self._pre_flight:
```

Comment:
If `self._pre_flight` is false, I think this is still shown in order to display that it was skipped - does this change that?

Reply:
this was back when I was trying to stick upgrade into work